src/cake.coffee

```diff
@@ -46,7 +46,7 @@ helpers.extend global,
 # asynchrony may cause tasks to execute in a different order than you'd expect.
 # If no tasks are passed, print the help screen. Keep a reference to the
 # original directory name, when running Cake tasks from subdirectories.
-exports.run = ->
+exports.run = helpers.async ->
```

Comment:
What comment?

I wanna hear from others before I change anything. I will remove the polyfill language from the comments because that is causing issues for you. This isn't really a polyfill, as the term "polyfill" only applies to web browsers. In this case `async` is simply just another helper function.